selenium_tests/lti_emailer/regression_suite.py

```diff
@@ -2,31 +2,36 @@
 import time
 from os import path, makedirs
 
-import HTMLTestRunner
+from selenium_common import HTMLTestRunner
 from selenium_tests.lti_emailer.emailer_permission_tests import EmailerPermissionTests
 
 
-date_timestamp = time.strftime('%Y%m%d_%H_%M_%S')
+class RegressionSuite(unittest.TestCase):
```

Comment:
I don't know where `main` came from, but it's not a standard `unittest.TestCase` method.  Mostly I don't understand why we're wrapping logic to bring up a loader, generate a test suite, and run tests, all in a class designed to have individual test cases on it.